projects/jazz-lib/src/lib/jazz-lib.module.ts

```diff
@@ -46,15 +50,18 @@ import {BreadcrumbsModule} from './breadcrumbs/breadcrumbs.module';
     ButtonComponent,
     HeaderComponent,
     MenuComponent,
+    MenuItemComponent,
+    MenuMainButtonComponent,
+    MenuSubButtonComponent,
     PagerComponent,
     ModalDialogComponent,
     SearchButtonComponent,
     UtilityItemComponent,
     UtilityMenuComponent,
     MainMenuItemComponent,
-    MainMenuComponent
+    MainMenuComponent,
+    TabComponent,
+    TabsComponent
   ]
 })
-export class SharedModule {
-
-}
+export class JazzLibModule { }
```

Comment:
DesignSystemLibModule instead of JazzLibModule.  We definitely want to keep the marketing name out of components.  The only exception to that will be the "jazz-" prefix that we are using on the tags.  The namespace on tags is more difficult to make unique because it is the namespace.  There is no other qualifier, and "jazz" was chosen because it appeared to be unique in the space (at least so far).